src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -49,17 +50,24 @@ public void parseCommand_clear() throws Exception {
     @Test
     public void parseCommand_delete() throws Exception {
         DeleteCommand command = (DeleteCommand) parser.parseCommand(
-                DeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
-        assertEquals(new DeleteCommand(INDEX_FIRST_PERSON), command);
+                DeleteCommand.COMMAND_WORD + " " + INDEX_FIRST.getOneBased());
+        assertEquals(new DeleteCommand(INDEX_FIRST), command);
     }
 
+    //    @Test
+    //    public void parseCommand_mates() throws Exception {
+    //        MatesCommand command = (MatesCommand) parser.parseCommand(
+    //                MatesCommand.COMMAND_WORD + " " + INDEX_FIRST.getOneBased());
+    //        assertEquals(new MatesCommand(INDEX_FIRST), command);
+    //    }
```

Comment:
You may need to override the `equals` method in `MatesCommand` first in order for this to work.

Reply:
Thank you!